.github/workflows/plgd-device-test.yml

```diff
@@ -21,23 +21,42 @@ jobs:
         include:
           - name: cloud-server-debug
             file: docker/apps/Dockerfile.cloud-server-debug
-            tag: cloud-server-debug:debug
+            tag: dbg
+            args: "CLOUD=1 SECURE=1 CREATE=1 MNT=1 DEBUG=1 OSCORE=0 ASAN=0 TSAN=0"
+          - name: cloud-server-debug-asan
+            file: docker/apps/Dockerfile.cloud-server-debug
+            tag: dbg
+            args: "CLOUD=1 SECURE=1 CREATE=1 MNT=1 DEBUG=1 OSCORE=0 ASAN=1 TSAN=0"
+          - name: cloud-server-debug-tsan
+            file: docker/apps/Dockerfile.cloud-server-debug
+            tag: dbg
+            args: "CLOUD=1 SECURE=1 CREATE=1 MNT=1 DEBUG=1 OSCORE=0 ASAN=0 TSAN=1"
           - name: cloud-server-discovery-resource-observable-debug
-            file: docker/apps/Dockerfile.cloud-server-discovery-resource-observable-debug
-            tag: cloud-server-discovery-resource-observable-debug:debug
-
+            file: docker/apps/Dockerfile.cloud-server-debug
+            tag: dbg
+            args: "CLOUD=1 SECURE=1 CREATE=1 MNT=1 DEBUG=1 OICRES_OBSERVABLE=1 OSCORE=0 ASAN=0 TSAN=0"
+          - name: cloud-server-discovery-resource-observable-debug-asan
```

Comment:
We had an agreement that at the moment we introduce more combinations, we won't use names but refactor it to tags as it's really messy.